src/EnergyPlus/RoomAirModelAirflowNetwork.cc

```diff
@@ -107,126 +106,65 @@ namespace RoomAirModelAirflowNetwork {
     // Do heat and moisture balance calculations on roomair nodes.
 
     // Using/Aliasing
-    using namespace DataRoomAirModel;
     using namespace DataHeatBalSurface;
     using namespace DataSurfaces;
     using namespace DataHeatBalance;
 
-    // Object Data
-
-    // Functions
-
-    void SimRoomAirModelAirflowNetwork(EnergyPlusData &state, int const ZoneNum) // index number for the specified zone
+    void SimRoomAirModelAFN(EnergyPlusData &state, int const zoneNum) // index number for the specified zone
     {
 
         // SUBROUTINE INFORMATION:
         //       AUTHOR         Brent Griffith
         //       DATE WRITTEN   January 2004/Aug 2005
         //       MODIFIED       Lixing Gu, Aug. 2015 for v8.4 replease
-        //       RE-ENGINEERED  na
 
         // PURPOSE OF THIS SUBROUTINE:
         // This subroutine manages RoomAirflowNetwork model simulation
 
         // METHODOLOGY EMPLOYED:
-        // calls subroutines
-
-        // REFERENCES:
-        // na
-
-        // USE STATEMENTS:
-        // na
-
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int ThisRoomAirNode;
-        int RAFNNum;
-
-        RAFNNum = state.dataRoomAirMod->RoomAirflowNetworkZoneInfo(ZoneNum).RAFNNum;
-
-        if (RAFNNum == 0) {
-            ShowFatalError(state,
-                           format("SimRoomAirModelAirflowNetwork: Zone is not defined in the RoomAirModelAirflowNetwork model ={}",
-                                  state.dataHeatBal->Zone(ZoneNum).Name));
-        }
+        // calls subroutines (LOL)
 
-        auto &thisRAFN(state.dataRoomAirflowNetModel->RAFN(RAFNNum));
```

Comment:
I'm not sure what the `RAFN` struct and array were supposed to accomplish other than making things seem to be object-oriented when they were in fact not, so I got rid of them.